File: app/src/main/java/hu/epam/worktime/mvvmworkdroid/modules/main/main/router/MainRouter.java
package hu.epam.worktime.mvvmworkdroid.modules.main.main.router;

import hu.epam.worktime.mvvmworkdroid.modules.services.models.WorkTime;

/**
 *
 *
 * Created by Mihaly_Hunyady on 2016. 12. 02..
 */

public interface MainRouter {
    void openDetails(WorkTime workTime);

    void openNewEntry();
}

File: app/src/main/java/hu/epam/worktime/mvvmworkdroid/modules/main/main/router/MainRouter.kt
package hu.epam.worktime.mvvmworkdroid.modules.main.main.router

import android.support.v4.util.Pair
import android.view.View
import hu.epam.worktime.mvvmworkdroid.modules.services.models.WorkTime

/**


 * Created by Mihaly_Hunyady on 2016. 12. 02..
 */

interface MainRouter {
    fun openDetails(sharedViews: Array<Pair<View, String>>, workTime: WorkTime)

    fun openNewEntry()
}
